BikeTrack/ContentService.swift

```diff
@@ -1,21 +1,20 @@
 import Foundation
 import os.log
-import Combine
 
 /**
  `ContentService` provides an interface into storing and retrieving user content.
  
  Being an observable object, `ContentService` is intended to be initialised as an `ObservableObject` high up in the SwiftUI view hierarchy and observed as an EnvironmentObject by child views as required.
  Like other `[Name]Service` classes, `ContentService` provides a layer of abstraction between the UI and lower level repositories, which can be swapped out at `init()`.
  */
+@MainActor
```

Comment:
I like this (and it led me down a rabbit hole) - 
Technically, any ObservableObject instantiated within a view will be marked as a main actor. However, from a code readability point of view, this gives some explanation to the reader as to the intent of the class, but also why we've replaced all `DispatchQueue.main.async {}` calls with `await MainActor.run {}`. [More detail here](https://www.hackingwithswift.com/quick-start/concurrency/how-to-use-mainactor-to-run-code-on-the-main-queue).

Reply:
Interestingly, when ObservedObjects are handled as main actors when instantiated in views, their async nature is also handled behind the scenes.

In our UI tests however - where the Service is instantiated within the test - it needs an `await` - 

```
let mockUserService = await UserService(authRepository: MockAuthRepository())

```